test/integration/unknown.cc

```diff
@@ -135,3 +135,54 @@ TEST(UnrecognizedElements, UnrecognizedElementsWithNamespaces)
   const auto errors = root.Load(testFile, config);
   EXPECT_TRUE(errors.empty());
 }
+
+/////////////////////////////////////////////////
+/// Test that elements that aren't part of the version of the spec
+/// cause an error (#327) - e.g. something that was valid in SDFormat 1.6 but
+/// not in SDFormat>=1.7 (https://git.io/JtKKb).
+TEST(UnrecognizedElements, OldElementsInNewSchemas)
+{
+  // This should be valid in 1.6, but not in 1.8 (do to usage of
+  // `use_parent_model_frame`).
+  auto make_model_xml_string = [](std::string version) {
```

Comment:
nit: our codecheck doesn't like the `{` on the same line

Reply:
Done. Oops!